tasks/create-advisory/create-advisory.yaml

```diff
@@ -65,6 +65,13 @@ spec:
         advisoryData=$(jq -c "$(params.jsonKey)" $(workspaces.data.path)/$(params.dataPath))
         configMapName=$(jq -er '.sign.configMapName' $(workspaces.data.path)/$(params.dataPath))
 
+        # Validate type
+        advisoryType=$(jq -c "$(params.jsonKey).type" $(workspaces.data.path)/$(params.dataPath))
```

Comment:
I think Martin's suggestion to use the advisoryData instead of reading it from the file again was a good one. Otherwise LGTM